draft-ietf-quic-recovery.md

```diff
@@ -130,7 +130,7 @@ important to the loss detection and congestion control machinery below.
   machinery of QUIC underneath.
 
 * ACK frames contain acknowledgment information.  QUIC uses a SACK-based
```

Comment:
Now that I read this, I don't think we want to use the word SACK here, since that has some baggage from TCP.  How about "ACK frames contain one or more ranges of acknowledged packets."?